docs/basics/userguide/seeding.md

```diff
@@ -349,7 +383,7 @@ run 3 :
 
 
 
-If you add to this code a [Seeder](rlberry.seeding.seeder.Seeder), use the `set_external_seed` method, and re-run it, you should have the same 'random' numbers everytime.
+If you add to this code a [Seeder](rlberry.seeding.seeder.Seeder), use the `set_external_seed` method, and re-run it, you should have the same 'random' numbers every time.
```

Comment:
Same: you don't have to spell it out that you should have the same random numbers, it is already seen in the result below.